### ISSUE TITLE
Mechanism references tab

### ISSUE THREAD

**Issue by Anneli15** [chembl/IDG#27]
Where does the filed "mech reference" get filled from? I put a box around it in the screen shot...
![screen shot 2015-06-29 at 4 33 51 pm](https://cloud.githubusercontent.com/assets/12811998/8411628/eeb3ebc2-1e7c-11e5-971f-dbe1c0473622.png)

**Comment by revajk** [comment]
it is a foreign key from the mechanism table.

**Comment by Anneli15** [comment]
Show me tomorrow, not sure I understand. Another question, is there place for a cancel option? I was going to add another synonym but changed my mind... I realized I can just move to next tab so maybe not necessary...

**Comment by Anneli15** [comment]
I've added a new drug with mechanism and started adding references. We kind of talked about this last week but it may have been left unanswered among all the other things we discussed. Looks like the 'mech reference' has been changed to mechanism type only. This would be fine but the target needs to be connected somehow as the reference isn't the same for the mechanism and all drugs but for a specific drug-target-mechanism relationship. At the moment, there are lots of references but there is nothing linking to/indicating which drug the reference is for? Maybe I'm missing something but i can't find this anywhere.
